buddyboss/buddyboss-platform
# PROD-698 #3126 - Fix: Links preview incorrect when we assign link to some strings.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3126 .

### How to test the changes in this Pull Request:

1. Go to the activity feed.
2. create the activity and add some text and add the link.
3. see the preview is correct now.

### Proof Screenshots or Video


https://vimeo.com/646280146/0bbae10043
### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
